### PR TITLE
Experiencing terraform

### PR DESCRIPTION
**Pull request by nschnepf**
First pass review/edit

[Original document](https://github.com/oracle-devrel/devo.tutorials/blob/main/tf-101/2-experiencing-terraform.md)

### CODE REVIEW

File: tf-101/2-experiencing-terraform.md [hunk 1]
```diff
@@ -174,241 +198,260 @@ should now work.
...
+
+Let's look at the plan that Terraform proposes. The `terraform plan` command provides a preview of the actions that Terraform will take to configure resources according to our configuration file.
+
+1. In a console, run:
+
+   ```console
+   terraform plan
+   ```
+
+     Terraform echoes something similar to the following:
```
Comment (ttscoff) [inline tf-101/2-experiencing-terraform.md:216]: The following grafs and code blocks are double indented, should only have one level of indentation

File: tf-101/2-experiencing-terraform.md [hunk 2]
```diff
@@ -174,241 +198,260 @@ should now work.
...
+
+   Destroy complete! Resources: 2 destroyed.
+   ```
+
+> **Fun facts:**  
+>
+> * It took Terraform under 10 seconds to provision a VCN and subnet (try it yourself by running `time terraform apply -auto-approve`), *and*
+> * under 7 seconds to destroy (try it yourself by running `time terraform destroy -auto-approve`) those same resources.  
+>  
+> Try to beat that doing it by hand in the OCI Console! {:.notice}
```
Comment (ttscoff) [inline tf-101/2-experiencing-terraform.md:436]: .notice on new line

**Review by ttscoff — CHANGES_REQUESTED** [review]

File: tf-101/2-experiencing-terraform.md [hunk 3]
```diff
@@ -174,241 +198,260 @@ should now work.
 
 If you ever set or change modules or backend configuration for Terraform,
 rerun this command to reinitialize your working directory. If you forget, other
-commands will detect it and remind you to do so if necessary.
-$ 
+commands will detect it and remind you to do so if necessary. 
 ```
```
[the remark this answers is not in the document]
Comment (nschnepf) [reply tf-101/2-experiencing-terraform.md:202]: The end of this code fence terminates a long block of output preceding it. It's not meant to start a new one. I don't believe a new line is required here.

[the remark this answers is not in the document]
Comment (ttscoff) [reply tf-101/2-experiencing-terraform.md:202]: Sorry, misread the output at the end of the code block as a paragraph, not console output. Ignore.

**Review by ttscoff — APPROVED** [review]

**Pull request merged**